### PR TITLE
Styling for all pages

### PR DESCRIPTION
**Pull request by nihabhaskar**
Styling changes we discussed on capstone project log.

### CODE REVIEW

**Comment by nihabhaskar** [comment]
Current Styling: https://granite-step-2020.uc.r.appspot.com/

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -8,30 +8,32 @@
...
-
   <script type="module" src="scripts/index-script.js" defer></script>
+  <link rel="stylesheet" href="styles/general.css">
   <script type="module" src="scripts/nav-bar.js"></script>
-
   <link rel="stylesheet" href="styles/homestyle.css">
 </head>
 <body>
-  <div id="nav-placeholder"> </div>
+  <div id="nav-bar">
```
Comment (rmr167) [inline src/main/webapp/index.html:17]: Will this div "nav-bar" be around the nav bar on all pages?

File: src/main/webapp/nav.html [hunk 2]
```diff
@@ -1,8 +1,9 @@
-
 <link rel="stylesheet" href="styles/general.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (rmr167) [inline src/main/webapp/nav.html:2]: Is the font-awesome import necessary here?  It is used for icons.

File: src/main/webapp/styles/general.css [hunk 3]
```diff
@@ -4,6 +4,9 @@
   --backColor: white;
   --mainColor: #4285f4;
   --placeholder: #b7b7b7;
+  --gRed: #db4437;
+  --gLGray: #bdbdbd;
+  --gDGray: #666;
```
Comment (rmr167) [inline src/main/webapp/styles/general.css:9]: This is just a question for my implementation of styling.  Can these global variables be used on any style sheet as long as general.css is imported on the same html page?

Comment (nihabhaskar) [reply src/main/webapp/styles/general.css:9]: yes

Comment (nihabhaskar) [reply src/main/webapp/styles/general.css:9]: just refer to them as var(--gLGray) for example

File: src/main/webapp/nav.html (hunk 2 above)
Comment (nihabhaskar) [reply src/main/webapp/nav.html:2]: i dont think so ill remove it

File: src/main/webapp/index.html (hunk 1 above)
Comment (nihabhaskar) [reply src/main/webapp/index.html:17]: yes it needs to be there for styling purposes

Comment (nihabhaskar) [reply src/main/webapp/index.html:17]: so everypage will need to have

  <div id="nav-bar">
    <div id="nav-placeholder"> </div>
  </div>

Comment (nihabhaskar) [reply src/main/webapp/index.html:17]: <div id="nav-bar">
    <div id="nav-placeholder"> </div>
  </div>

**Pull request merged**